glidex.forms.sample/Forms/ImageResourceExtension.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace TestingApp
+{
+    [ContentProperty(nameof(Source))]
+    public class ImageResourceExtension : IMarkupExtension
+    {
+        public string Source { get; set; }
+
+        public object ProvideValue(IServiceProvider serviceProvider)
+        {
+            if (Source == null)
+            {
+                return null;
+            }
+
+            // Do your translation lookup here, using whatever method you require
+            var imageSource = ImageSource.FromResource(Source, typeof(ImageResourceExtension).GetTypeInfo().Assembly);
```

Comment:
`ImageResourceExtension` is in this assembly, `glidex.forms.dll`. It will never have any `EmbeddedResource` files inside? How would this code ever load an image?

Reply:
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/images?tabs=windows

ImageResourceExtension is a class recommended by Microsoft to return the image source of an Embedderd Resource image.

Reply:
But how does it work to include it in `glidex.forms.dll`? `typeof(ImageResourceExtension).GetTypeInfo().Assembly` will never have `EmbeddedResource` inside.

Reply:
U dont include it in the dll , u include in the project as a normal class.
Sorry i told you i am so new to git to i dont know where i added it to the repo.